### PR TITLE
1kG examples: add w/s 'dashboard' file, and some notebook cleanup

### PR DESCRIPTION
**Pull request by amygdala**
... fix typo, set nb kernel to R, 'terra'--> 'wb'

(for the notebook, the bulk of it is drawn from another source which I’m leaving it as is, so you just need to look at the changes)

### CODE REVIEW

File: 1kgenomes_examples/dashboard.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+
+# 1000 Genomes examples
+
+## Introduction
+
+This workspace collects several example notebooks that use the 1000 Genomes public dataset.
+
+To run the examples, **first Duplicate this workspace**.  You can do this via the 'three-dot' menu
```
Comment (emmarogge) [inline 1kgenomes_examples/dashboard.md]:
```suggestion
To run the examples, **first duplicate this workspace**.  You can do this via the 'three-dot' menu
```

File: 1kgenomes_examples/dashboard.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+next to the 'Share' button at the top right in the Workbench UI.
+
+Then, in your new workspace, create a JupyterLab R-based cloud environment.  You can take the defaults for
```
Comment (emmarogge) [inline 1kgenomes_examples/dashboard.md]:
```suggestion
Then, in your new workspace, create a JupyterLab R-based cloud environment.  You can accept the defaults for
```

File: 1kgenomes_examples/dashboard.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+machine size and number of CPUs. No GPUs are necessary.
+
+Click the link to open the JupyterLab server once the cloud environment is up and running. In the
+left-hand file navigator, click in to `repos/terra-axon-examples/1kgenones_examples`.  In this
+directory you'll see three example notebook files, described below.
+
+When you run the R examples (`R_1k_genomes.ipynb` and `1kg_analysis.ipynb` ), ensure that you select
+the R kernel. For `GWAS_experiments.ipynb`, use the Python 3 kernel.
```
Comment (emmarogge) [inline 1kgenomes_examples/dashboard.md:19]: Could we provide a hyperlink to instructions for switching from one kernel to another?

File: 1kgenomes_examples/dashboard.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+## The example notebooks
+
+### R Single Chromosome PCA Example: Reading variant data into an R sparse matrix
+
+The **`R_1k_genomes.ipynb`** notebook is adapted from:  \
+http://bwlewis.github.io/1000_genomes_examples/PCA.html  \
+https://github.com/bwlewis/1000_genomes_examples
+
+This example walks through the computation of principal components (PCA) of genomic variant data
+across one chromosome from 2,504 people from the [1000 genomes
```
Comment (emmarogge) [inline 1kgenomes_examples/dashboard.md]:
```suggestion
across one chromosome from 2,504 people from the [1000 Genomes
```

File: 1kgenomes_examples/dashboard.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+variant data for one chromosome into a three-dimensional subspace, and then plots the result.
+
+The example uses:
+
+- a very simple C parsing program to efficiently read variant data into an R sparse matrix,
+- the `irlba` package to efficiently compute principal components,
+- the `threejs` package to visualize the result.
+
+This example is intended to be run in a Verily Workbench notebook cloud environment ('Jupyterlab
+Vertex AI Workbench instance'), using the R environment image.  You can take the defaults when
```
Comment (emmarogge) [inline 1kgenomes_examples/dashboard.md]:
```suggestion
Vertex AI Workbench instance'), using the R environment image.  You can accept the defaults when
```

**Review by emmarogge — APPROVED** [review]
LGTM!

File: 1kgenomes_examples/dashboard.md (hunk 3 above)
Comment (amygdala) [reply 1kgenomes_examples/dashboard.md:19]: Good idea, I will make a note to find something for the next iteration.  Now that I fixed the nb to automatically use the R kernel it's not quite so urgent.

**Comment by amygdala** [comment]
thank you, @emmarogge !

**Pull request merged**